client/modules/IDE/components/Editor.jsx

```diff
@@ -269,17 +269,21 @@ class Editor extends React.Component {
       indent_size: INDENTATION_AMOUNT,
       indent_with_tabs: IS_TAB_INDENT
     };
-
+    console.log('inside');
```

Comment:
Could you remove the `console.log` statements?

Reply:
thanks for mentioning i have made a commit that removes it